src/adapters/Cornerstone/Segmentation_4X.js

```diff
@@ -629,6 +629,43 @@ function checkSEGsOverlapping(
     var groupsLen = PerFrameFunctionalGroupsSequence.length;
     var numberOfSegs = multiframe.SegmentSequence.length;
     var numberOfFrames = imageIds.length;
+    // NOTE:
+    // Assumption1 : numberOfSegs * numberOfFrames = groupsLen,
+    // i.e. for each frame we have a N PerFrameFunctionalGroupsSequence, where N is the numberOfSegs.
+    // Assumption2 : the order of the group sequence is = numberOfFrames of segmentation 1 +  numberOfFrames of segmentation 2 + ... + numberOfFrames of segmentation numberOfSegs
+    // -------------------
+    // We could check the ImagePositionPatient and working in 3D coordinates (instead of indexes) and remove the assumptions,
+    // but this would greatly increase the computation time (i.e. we would have to sort the data before running checkSEGsOverlapping).
+
+    if (numberOfSegs * numberOfFrames !== groupsLen) {
+        console.warn(
+            "checkSEGsOverlapping: it not possible performing the overlapping segments check since PerFrameFunctionalGroupsSequence has missing data."
```

Comment:
PerFrameFunctionalGroupsSequence is type 1, maybe we should just break or define a new message?
Failed to perform overlap of segments: missing PerFrameFunctionalGroupsSequence